.github/workflows/build.yml

```diff
@@ -73,8 +96,15 @@ jobs:
   build-stellar-soroban-rpc:
     runs-on: ubuntu-latest
     steps:
-    - uses: docker/setup-buildx-action@5146db6c4d81fbfd508899f851bbb3883a96ff9f
-    - name: Build Stellar-Soraban-Rpc Image
+    - id: cache
+      uses: actions/cache@v3
+      with:
+        path: /tmp/image
+        key: image-soroban-rpc-${{ env.CORE_REPO_BRANCH }}
```

Comment:
should this key be serialized with `env.SOROBAN_TOOLS_REPO_BRANCH` rather than `env.CORE_REPO_BRANCH` for uniqueness?

Reply:
Thanks, this is a typo. Fixed in cffe0da.